src/tests/ftest/object/same_key_different_value.py

```diff
@@ -71,13 +59,12 @@ def test_single_to_array_value(self):
         for i in range(3):
             try:
                 # create an object and write single value data into it
-                obj = self.container.write_an_obj(single_value_data,
-                                                  len(single_value_data)+1,
-                                                  dkey, akey, obj_cls=1)
+                obj = self.container.write_an_obj(
```

Comment:
(pylint-no-member) Instance of 'list' has no 'write_an_obj' member
(pylint-no-member) Instance of 'TestContainer' has no 'write_an_obj' member